### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by dan9186**
* correcting some badges
* fixing deploy

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,9 +54,9 @@ coverage_compfriendly:  ## Generates the code coverage in a computer friendly ma
 
 .PHONY: crosscompile
 crosscompile:  ## Build the binaries for the primary OS'
-	GOOS=linux $(GOBUILD) -ldflags "-X main.buildTime=$(DATE) -X main.appVersion=$(BUILD_VERSION)" -o $(APP)-linux .
-	GOOS=darwin $(GOBUILD) -ldflags "-X main.buildTime=$(DATE) -X main.appVersion=$(BUILD_VERSION)" -o $(APP)-darwin .
-	GOOS=windows $(GOBUILD) -ldflags "-X main.buildTime=$(DATE) -X main.appVersion=$(BUILD_VERSION)" -o $(APP)-windows.exe .
+	GOOS=linux $(GOBUILD) -ldflags "-X main.buildTime=$(DATE) -X main.appVersion=$(BUILD_VERSION)" -o deploy/linux/$(APP) .
```
Comment (kellyp) [inline Makefile]: I think the service make files need a `bin` directory after the os name:
```https://s3.amazonaws.com/public.ionchannel.io/files/ionize/linux/bin/ionize```

Comment (dan9186) [reply Makefile]: I wondered about that too, but the previous step that was working didn't seem to have that https://github.com/ion-channel/ionize/commit/9e07883c845bb4ca8303d0c91fa5c9a7d6c3fde1#diff-b67911656ef5d18c4ae36cb6741b7965L64

Comment (dan9186) [reply Makefile]: Now I'm wondering how it ever worked.

Comment (kellyp) [reply Makefile]: i'm not sure it ever worked i did a manual deploy to the .../bin/ dir.

Comment (dan9186) [reply Makefile]: Well then, let's kick this pig

**Pull request merged**